src/styles/_patterns_card-deck.scss

```diff
@@ -0,0 +1,93 @@
+@mixin p-card-deck {
```

Comment:
It's best practise to name space your site specific mixins with a site identifier. For example `sw-p-card-deck`. This helps to stop naming collisions.